### PR TITLE
Add decrement button for count

### PR DESCRIPTION
**Pull request by TahsinAyman**
This pull request adds a decrement button for the count in the App component. Users can now decrease the count by clicking on the "-" button. This feature enhances the functionality of the app and provides a better user experience.

No issues were fixed in this pull request.

### CODE REVIEW

**Review by fullstackltd — DISMISSED** [review]
You have a tiger

File: src/App.tsx [hunk 1]
```diff
@@ -21,6 +21,9 @@ function App() {
         <button onClick={() => setCount((count) => count + 1)}>
           count is {count}
         </button>
+        <button onClick={() => setCount((count) => count - 1)}>
+          -
+        </button>
```
Comment (fullstackltd) [inline src/App.tsx:26]: Change the text to "Decrement'

**Review by fullstackltd — CHANGES_REQUESTED** [review]
please see the comments on the code

File: src/App.tsx (hunk 1 above)
Comment (TahsinAyman) [reply src/App.tsx]: done working on it

**Review by fullstackltd — APPROVED** [review]
well done

**Pull request merged**